frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -98,8 +101,12 @@ def add_webhook_history_record(webhook_uuid, user_agent='Not Set',
     db.session.add(webhookRecord)
     db.session.commit()
 
+    if not isinstance(builds_initiated_via_hook, list):
+        builds_initiated_via_hook = [builds_initiated_via_hook]
+
     for build in builds_initiated_via_hook:
         build.webhook_history_id = webhookRecord.id
+        db.session.commit()
```

Comment:
you can avoid this commit() actually;  you just have to avoid working with IDs directly, but use the references..

Reply:
example https://github.com/fedora-copr/copr/blob/916f82f059610c2af320d01b1893d59ffd27c725/frontend/coprs_frontend/coprs/logic/coprs_logic.py#L333

Reply:
- nice I dont need to use any `commit()` in this method after using refs. 
- renamed `webhookRecord` to `webhook_record` (JS instincts may have kicked in :D )